README.md

```diff
@@ -28,19 +38,29 @@ element.innerHTML = 'Hello, DOM!'
 element.style.backgroundColor = '#f9f9f9';
 ```
 
-Feel free to set as many properties as you'd like — this is a good chance to look around and explore different properties of DOM elements!
+Feel free to set as many properties as you'd like — this is a good chance to
+look around and explore different properties of DOM elements!
+
+But notice that no matter what properties we add, the element doesn't show up on
+the page. What gives?
 
-But notice that no matter what properties we add, the element doesn't show up on the page. What gives?
+## Append Elements into the DOM
 
-## `appendChild()`
+To get an element to appear in the DOM, we have to append it to an existing DOM
+node. We can start as high up on the tree as `document.body`, or we can find a
```

Comment:
To go back to our tree metaphor, we have to glue our new leaf onto a branch that's already there.